flow-typed/npm/mocha_v5.x.x.js

```diff
@@ -37,16 +37,25 @@ declare interface $npm$mocha$Suite {
 }
 
 declare interface $npm$mocha$ContextDefinition {
-  (description: string, callback: (/* this: $npm$mocha$SuiteCallbackContext */) => void): $npm$mocha$Suite;
-  only(description: string, callback: (/* this: $npm$mocha$SuiteCallbackContext */) => void): $npm$mocha$Suite;
-  skip(description: string, callback: (/* this: $npm$mocha$SuiteCallbackContext */) => void): void;
+  (description: string, callback: () => /* this: $npm$mocha$SuiteCallbackContext */ void): $npm$mocha$Suite;
+  only(description: string, callback: () => /* this: $npm$mocha$SuiteCallbackContext */ void): $npm$mocha$Suite;
+  skip(description: string, callback: () => /* this: $npm$mocha$SuiteCallbackContext */ void): void;
```

Comment:
This is an interesting change. Maybe worth a PR against [flow-typed](https://github.com/flow-typed/flow-typed)? Not sure of how correct this is yet